vra/resource_project.go

```diff
@@ -179,16 +186,16 @@ func resourceProjectRead(d *schema.ResourceData, m interface{}) error {
 		return err
 	}
 	project := *ret.Payload
-	d.Set("administrators", flattenUserList(project.Administrators))
+	d.Set("administrators", flattenUsers(project.Administrators))
 	d.Set("constraints", flattenProjectConstraints(project.Constraints))
 	d.Set("custom_properties", project.CustomProperties)
 	d.Set("description", project.Description)
 	d.Set("machine_naming_template", project.MachineNamingTemplate)
-	d.Set("members", flattenUserList(project.Members))
+	d.Set("members", flattenUsers(project.Members))
 	d.Set("name", project.Name)
 	d.Set("operation_timeout", project.OperationTimeout)
 	d.Set("shared_resources", project.SharedResources)
-	d.Set("viewers", flattenUserList(project.Viewers))
+	d.Set("viewers", flattenUsers(project.Viewers))
 	d.Set("zone_assignments", flattenZoneAssignment(project.Zones))
```

Comment:
We should set the administrator_roles, member_roles and viewer_roles in the read method to reflect in the state file

Reply:
Done